add APK_MIN_SDK_VERSION to KB (#94)
Ostorlab/KB: MOBILE_CLIENT/ANDROID/_LOW/APK_MIN_SDK_VERSION/description.md

> The `android:minSdkVersion` attribute specifies the minimum Android API level required by the application. Setting a low `minSdkVersion` may allow the application to run on older Android versions but could expose users to security vulnerabilities.

Reviewer: ```suggestion
The `android:minSdkVersion` attribute specifies the minimum Android API level required by the application. Setting a low `minSdkVersion` may allow the application to run on older Android versions which can expose users to security vulnerabilities.
```

> The `android:minSdkVersion` attribute specifies the minimum Android API level required by the application. Setting a low `minSdkVersion` may allow the application to run on older Android versions but could expose users to security vulnerabilities.

Reviewer 2: Here is a link with the deprecation notice:
https://support.google.com/googleplay/android-developer/answer/11926878?hl=en
